practice/react-app/src/App.tsx

```diff
@@ -1,35 +1,46 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import { useState } from 'react';
+import { operations, MathOperation } from './mathOperations';
+import { Button } from './OperationButton';
+import { OperationsPanel } from './OperationPanel';
+import { NumericInput } from './NumericInput';
 
-function App() {
-  const [count, setCount] = useState(0)
+const App = () => {
+  const [numA, setNumA] = useState<number>(0);
+  const [numB, setNumB] = useState<number>(0);
+  const [result, setResult] = useState<number | string>(0);
+
+  const setNumber = (
+    func: React.Dispatch<React.SetStateAction<number>>,
+    value: number
+  ) => {
+    func(value);
+  };
+
+  const doWork = (func: MathOperation) => {
```

Comment:
Ten "doWork" dość generyczny - może warto precyzyjnie określić jakiego rodzaju to praca?